lib/debug/breakpoint.rb

```diff
@@ -64,6 +64,11 @@ def description
       to_s
     end
 
+    def duplicable?
+      # only binding.bp or DEBUGGER__.console are duplicable
+      is_a?(LineBreakpoint) && oneshot
```

Comment:
Could you use inheritance?

define false and define it in `LineBreakpoint`.

Reply:
sounds good. will do 👍